tests/autorestart_test/test_container_autorestart.py

```diff
@@ -182,20 +182,12 @@ def kill_process_by_pid(duthost, container_name, program_name, program_pid):
     """
     @summary: Kill a process in the specified container by its pid
     """
-    duthost.shell("docker exec {} kill -SIGKILL {}".format(container_name, program_pid))
+    kill_cmd_output = duthost.shell("docker exec {} kill -SIGKILL {}".format(container_name, program_pid))
```

Comment:
Suggest renaming to `kill_cmd_result`

Reply:
Will update and the exit code is the result of `kill` command not the output of this command.